### DISCUSSION TITLE
Voice Channel None but user is in voice

### DISCUSSION THREAD

**Discussion by Kadethra** [Rapptz/discord.py#8305] [Q&A]
I just tried a bit Voice Channel stuff and encountered a Problem.
```
if not ctx.author.voice:
      return None
channel = ctx.author.voice.channel
```
This is currently the way I check if a user is in Voice or not. While testing I just kept being in Voice. When I restarted the Bot and tried to let it join, it got `ctx.author.voice` but it didn't get a channel (was None) even though I was in a Voice Channel.
Is there a way to fix it or is it just a bug?

**Comment by Rapptz** [comment]
This was fixed a few commits ago, are you up to date?

**Reply by Kadethra** [reply]
Nope I wasn't.. Thx! Should have tried it before.
